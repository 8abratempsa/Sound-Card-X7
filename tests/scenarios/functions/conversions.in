octosql "SELECT int(true), int(false)"
