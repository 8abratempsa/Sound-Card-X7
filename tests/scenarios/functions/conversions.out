+-------+-------+
| col_0 | col_1 |
+-------+-------+
|     1 |     0 |
+-------+-------+
